apistar/server/injector.py

```diff
@@ -0,0 +1,108 @@
+from inspect import Parameter, signature
+
+from apistar.exceptions import ConfigurationError
+
+
+class Component():
+    resolves = None
+
+    def identity(self, parameter: Parameter):
+        """
+        Each component needs a unique identifier string that we use for lookups
```

Comment:
perhaps mention that it is a unique, case-insensitive name?

Reply:
Thanks for the review! The name isn't actually case-insensitive, I'm just using the lowercase for stylistic reasons.